torch/csrc/jit/python/pybind_utils.h

```diff
@@ -496,17 +496,43 @@ inline InferredType tryToInferType(py::handle input) {
   }
 
   // Try container types
-  return tryToInferContainerType(input);
+  return tryToInferContainerType(input, false);
 }
 
-inline InferredType tryToInferContainerType(py::handle input) {
+// This function is similar to tryToInferType, but it only tries to infer
+// primitive types (int, float, bool, complex).
```

Comment:
Can we document this in the profiler docs for this feature?

Reply:
Can you send me the link to the profiler docs?

Reply:
I will update https://www.internalfb.com/intern/wiki/PyTorch/Using_PyTorch/PyTorch_Execution_Trace/ after PR is landed